how-to/workspace-platform-starter/client/src/framework/apps.ts

```diff
@@ -88,6 +89,10 @@ export async function init(
 			cacheDuration += options?.cacheDurationInMinutes * 60 * 1000;
 		}
 		supportedManifestTypes = options?.manifestTypes ?? [];
+
```

Comment:
Is it possible to have this backed by a setTimeout that calls itself after it runs so that there is only ever one job queued on the event loop. Someone might put a very low cache timeout and there may be a number of app directories (plus we listen for connected apps that might have a delay). setInterval will add a job to the event loop every e.g. 1 second (if set to that) even if the provider is busy. Should this behaviour be configurable?